less/v2/character.less

```
@@ -414,6 +420,11 @@
               }
             }
           }
+
+          .lozenges.no-margin-top {
+            margin-top: 0px;
+          }
+
```

Would prefer to avoid adding a class just for this styling.

```suggestion
          .top:has([data-n="7"]) + .lozenges { margin-block-start: 0; }
```